assets/src/components/PostMeta/withPostMeta.js

```diff
@@ -57,6 +57,7 @@ export function withPostMeta( WrappedComponent ) {
         )
       ) {
         value = this.props.defaultValue;
+        this.handleChange( metaKey, value );
```

Comment:
This definitely should not happen in `render`, as it will result in another render, and in general `render` should not have any side effects, even if they would not trigger a render.

Did you do this to trigger `this.props.writeMeta`or the preview update, or both?

Reply:
Resolving this as it changed and it's outdated now.